vertexai/genai/client.go

```diff
@@ -95,6 +95,7 @@ type GenerativeModel struct {
 	GenerationConfig
 	SafetySettings []*SafetySetting
 	Tools          []*Tool
+	ToolConfig     *ToolConfig
```

Comment:
This requires a docstring and an example.

Reply:
Did both.